repo/version.go

```diff
@@ -1,9 +1,9 @@
-package config
```

Comment:
This doesn't really feel like the right place to put this. Personally, I'd put it in the root along side `doc.go`.